### PR TITLE
docs: reworked Lima onboarding

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

docs: reworked Lima onboarding

- Split the procedure: _Creating a Lima machine for container workloads with Podman Desktop_ and - _Creating a Lima machine for Kubernetes workloads with Podman Desktop_
- Review for language, minimalism, modularity.

### What issues does this PR fix or reference?

references #3790

### CODE REVIEW

**Comment by afbjorklund** [comment]
They are called "instances" in the Lima documentation, not sure if it is because "lima" has machine in the name already ?

```console
$ limactl ls --help
List instances of Lima.

...
```

The main goal was to make the documentation consistent with https://lima-vm.io/docs/ when it came to terminology

File: website/docs/onboarding/containers/creating-a-lima-instance-with-podman-desktop.md [hunk 1]
```diff
@@ -20,79 +20,52 @@ Consider creating a custom Lima instance to:
...
 
-To create an instance "docker" from a template "docker":
+     ```shell-session
+     $ limactl start --name=podman template://podman-rootful
+     ```
 
-```console
-$ limactl start --name=docker template://docker
-```
+   - To create an Lima machine with Docker, use the `docker` template:
```
Comment (afbjorklund) [inline website/docs/onboarding/containers/creating-a-lima-instance-with-podman-desktop.md]: with rootless Docker (same as above)

File: website/docs/onboarding/containers/creating-a-lima-instance-with-podman-desktop.md [hunk 2]
```diff
@@ -20,79 +20,52 @@ Consider creating a custom Lima instance to:
...
+  $ podman.lima version
+  ```
 
-You can also use the `kubectl.lima` wrapper script, to connect to the cluster:
+- To verify the connection to a running "docker" instance:
 
-```console
-$ kubectl.lima version
-```
+  ```cshell-session
```
Comment (afbjorklund) [inline website/docs/onboarding/containers/creating-a-lima-instance-with-podman-desktop.md]: typo ? shell-session

**Comment by afbjorklund** [comment]
I prefer using the ".lima" wrappers, over having to remember location of sockets and configs...

https://lima-vm.io/docs/examples/#running-containers

Note: Podman Desktop does **not** support containerd and `nerdctl`, except as a container runtime

So we don't mention `nerdctl.lima` or the `default` instance.

File: website/docs/onboarding/containers/creating-a-lima-instance-with-podman-desktop.md (hunk 1 above)
Comment (themr0c) [reply website/docs/onboarding/containers/creating-a-lima-instance-with-podman-desktop.md]:
```suggestion
   - To create an Lima machine with rootless Docker, use the `docker` template:
```

File: website/docs/onboarding/containers/creating-a-lima-instance-with-podman-desktop.md (hunk 2 above)
Comment (themr0c) [reply website/docs/onboarding/containers/creating-a-lima-instance-with-podman-desktop.md]:
```suggestion
  ```shell-session
```

**Comment by themr0c** [comment]
Oh my mistake then. I thought they were called Lima machines. I will fix that tomorrow.

**Comment by afbjorklund** [comment]
It is not important, I was just trying to explain why that word was used...

We have called them docker machines for a decade, and podman machines for (almost) 5 years. And now they're just Linux machines

**Review by benoitf — APPROVED** [review]
LGTM

probably need to have approval from @afbjorklund

**Review by afbjorklund — APPROVED** [review]
Looks good to me.

You might want to also add "rootful Docker" for completeness, but since the main focus is on Podman it might not be required to do so. But it would make it more "symmetrical"...

**Review by cdrage — APPROVED** [review]

File: website/docs/onboarding/containers/creating-a-lima-instance-with-podman-desktop.md [hunk 3]
```diff
@@ -20,79 +20,52 @@ Consider creating a custom Lima instance to:
...
 
-```console
-$ limactl start --name=docker template://docker
-```
+   - To create an Lima machine with rootless Docker, use the `docker` template:
 
-To select the number of CPUs and the memory or disk size:
+     ```shell-session
+     $ limactl start --name=docker template://docker
+     ```
```
Comment (afbjorklund) [inline website/docs/onboarding/containers/creating-a-lima-instance-with-podman-desktop.md:50]: - To create an Lima machine with rootless Docker, use the `docker` template:
     ```shell-session
     $ limactl start --name=docker template://docker
     ```
   - To create a Lima machine with rootful Docker, use the `docker-rootful` template:
     ```shell-session
     $ limactl start --name=docker template://docker-rootful
     ```

Comment (themr0c) [reply website/docs/onboarding/containers/creating-a-lima-instance-with-podman-desktop.md:50]: added

**Comment by themr0c** [comment]
@afbjorklund I apologize for the delay. I implemented your suggestions. Can you have a last look before we merge?

**Review by afbjorklund — APPROVED** [review]
Looks good!

**Comment by afbjorklund** [comment]
In a follow up PR (or not), we might want to document how to select which provider and which instance to use.

It will default to "podman", but if you use "docker" or "kubernetes" (instance "k3s" or "k8s") you have to change it.


It is currently located under "Preferences: Lima", it is a simple drop-down - but might be tricky for the user to find:

pd-lima-extension-podman
pd-lima-extension-docker
pd-lima-extension-k8s

For the kubernetes addition, one also has to modify the kubeconfig search path - until that can be automated.

----

All of it is somewhat temporary, the best for the default path would be to create the machines like with podman.

* https://github.com/containers/podman-desktop/issues/3069

Then the user would decided if they want to run "containers" or run "kubernetes", with all the other VM options:

* https://github.com/containers/podman-desktop/pull/2902

**Comment by afbjorklund** [comment]
Note: the changes suggested above are not connected to _this_ PR, about splitting the current page into two.

**Comment by themr0c** [comment]
OK, nice heads-up. I created a follow-up issue, and am merging this one.

**Pull request merged**